src/Wrapper.tsx

```diff
@@ -42,7 +42,7 @@ import UserAvatar from './common/UserAvatar'
 import Copyright from './Copyright'
 import { lightTheme } from './theme'
 
-const drawerWidth: number = 240
+const drawerWidth = 240
```

Comment:
Is there a reason the type's been removed?

Reply:
We're instantiating the var with a number, so it's already typed as a number. Giving it the type of `number` explicitly does nothing. Our eslint rules state that we shouldn't be doing this, so it's been removed by the auto-fixing.